lxd/network/openvswitch/ovs.go

```diff
@@ -373,3 +374,151 @@ func (o *OVS) OVNSouthboundDBRemoteAddress() (string, error) {
 
 	return addr, nil
 }
+
+// Default STP priority is 32768 (0x8000). This value will be used if the
+// priority cannot be retrieved or if STP is disabled
+func GetStpPriority(bridgeName string) (string, error) {
+	output, err := shared.RunCommand("ovs-appctl", "stp/show", bridgeName)
```

Comment:
I think `RunCommand` is deprecated. Instead you can use `RunCommandContext` and pass a `context.Context` into this method.